test/jmap/email/search/use_case_test/search_by_form_use_case_test.dart

```diff
@@ -0,0 +1,180 @@
+          "sessionState": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
+          "methodResponses": [
+            [
+              "Email/query",
+              {
+                "accountId":
+                "0eacc7a5c74b27ab36a823bc5c34da36e16c093705f241d6ed5f48ee73a4ecfb",
+                "queryState": "c4a08240",
+                "canCalculateChanges": false,
+                "ids": ["04f27c50-e879-11ec-aae4-43ebf0340ebd"],
+                "position": 0,
+                "limit": 256
+              },
+              "c1"
+            ],
+            [
+              "Email/get",
+              {
+                "accountId":
+                "0eacc7a5c74b27ab36a823bc5c34da36e16c093705f241d6ed5f48ee73a4ecfb",
+                "notFound": [],
+                "state": "94bbff20-e87c-11ec-aae4-43ebf0340ebd",
+                "list": [
+                  {
+                    "preview": "AAAA",
+                    "hasAttachment": false,
+                    "size": 3328,
+                    "subject": "AAAA",
+                    "from": [
+                      {
+                        "name": "Manh tuan Manh",
+                        "email": "manh199672@gmail.com"
+                      }
+                    ],
+                    "sentAt": "2022-06-10T04:44:03Z",
+                    "id": "04f27c50-e879-11ec-aae4-43ebf0340ebd",
+                    "receivedAt": "2022-06-10T04:51:41Z"
+                  }
+                ]
+              },
+              "c2"
+            ]
+          ]
+        }),
+        data: {
+          "using": ["urn:ietf:params:jmap:core", "urn:ietf:params:jmap:mail"],
+          "methodCalls": [
+            [
+              "Email/query",
+              {
+                "accountId":
+                "0eacc7a5c74b27ab36a823bc5c34da36e16c093705f241d6ed5f48ee73a4ecfb",
+                "filter": {"from": "manh"}
+              },
+              "c1"
+            ],
+            [
+              "Email/get",
+              {
+                "accountId":
+                "0eacc7a5c74b27ab36a823bc5c34da36e16c093705f241d6ed5f48ee73a4ecfb",
+                "#ids": {
+                  "resultOf": "c1",
+                  "name": "Email/query",
+                  "path": "ids/*"
+                },
+                "properties": [
+                  "id",
+                  "subject",
+                  "size",
+                  "from",
+                  "receivedAt",
+                  "sentAt",
+                  "preview",
+                  "hasAttachment"
+                ]
+              },
+              "c2"
+            ]
+          ]
+        },
+        headers: {
+          "accept": "application/json;jmapVersion=rfc-8621",
+          "content-type": "application/json; charset=utf-8"
+        });
+
+    final httpClient = HttpClient(dio);
+    final processingInvocation = ProcessingInvocation();
+    final jmapRequestBuilder =
+    JmapRequestBuilder(httpClient, processingInvocation);
```

Comment:
Should write `final jmapRequestBuilder = JmapRequestBuilder(httpClient, processingInvocation);`